mantidimaging/gui/windows/operations/filter_previews.py

```diff
@@ -193,6 +197,17 @@ def add_difference_overlay(self, diff):
         lut = map.getLookupTable(0, 1, 2)
         self.image_after_overlay.setLookupTable(lut)
```

Comment:
Could you rename `image_after_overlay` to `image_diff_overlay` to make things clearer now that there are 2 overlays.